Livecheckables/vramsteg.rb

```diff
@@ -0,0 +1,4 @@
+class Vramsteg
+  livecheck :url   => "https://github.com/GothenburgBitFactory/vramsteg.git",
+            :regex => /v?(\d+(?:\.\d+)+)/
```

Comment:
```suggestion
            :regex => /^v?(\d+(?:\.\d+)+)$/
```

You forgot the trailing `$`, so this would match pre-release versions as well. Leaving off the leading `^` means it can match unrelated tags like `somethingelse-1.2.3`, which I've often seen in other Git repos. Unless there's a compelling reason to modify the standard regex for Git tags, we should simply use it unmodified.

Reply:
👍 works for me!